docs/general/rendering.md

```diff
@@ -144,6 +144,32 @@ That's the result:
 
 This feature is specific to JsonApi, so you have to use the use the [JsonApi Adapter](adapters.md#jsonapi)
 
+
+##### Resource-level
+
+In your serializer, define each link in one of the following methods:
+
+As a static string
+
+```ruby
+link :link_name, 'https://example.com/resource'
+```
+
+As a block to be evaluated. When using Rails, URL helpers are available.
```

Comment:
I figured when Grape is used with Rails, the Railtie would load and SerializationContext would be available and include the url helpers. When Grape is used without Rails, the railtie would not load, SerializationContext would not be available, and no url helpers would be available. Are you wanting more of an explicit test? It doesn't look like links are even tested under Grape.